src/component/feature/form/form-update-feature-container.jsx

```diff
@@ -42,13 +41,13 @@ const prepare = (methods, dispatch) => {
         // TODO: should add error handling
         requestUpdateFeatureToggle(input)(dispatch)
             .then(() => methods.clear())
-            .then(() => hashHistory.push(`/features`));
+            .then(() => ownProps.history.push(`/features`));
```

Comment:
I am not sure if this is the right response to an update, but want to keep that in a separate issue, see: https://github.com/Unleash/unleash-frontend/issues/139